.github/workflows/ci-gh.yml

```diff
@@ -24,6 +24,9 @@ jobs:
           - release
         upload-enabled:
           - false
+        python-version:
+          #TODO cover the whole python and cuda matrix
+          - 3.12
```

Comment:
Is this choice binding? make_ci_env() does not seem to honor this choice. We can perhaps address this requirement in a later PR since it seems conda/mamba selects python 3.12 for us anyway: https://github.com/NVIDIA/cuda-python/actions/runs/11917344519/job/33212174669#step:4:588
?

Reply:
Please see my comment on make-conda-env.